source/codegen/metadata/nixnetsocket/custom_proto.mako

```diff
@@ -24,3 +24,19 @@ message SockOptData {
     string data_string = 4;
   }
 }
+
+message AddrInfo {
+  int32 flags = 1;
```

Comment:
I'm guessing a lot of these will be enums? ( @danestull ). Either way, we're probably going to need to take a few passes at enums to get everything right.

Reply:
Should be the enum `GetAddrInfoFlags` which is added in my change: https://github.com/ni/grpc-device/pull/610

Reply:
It seems correct that the four main fields here should be updated to use the new Enums. For both `AddrInfo` and `AddrInfoHint`, I updated fields 1-4 for to use the Enums `GetAddrInfoFlags`, `AddressFamily`, `SocketProtocolType`, and `IPProtocol`.